pkg/cri/resource-manager/cache/container.go

```diff
@@ -811,6 +811,15 @@ func (c *container) GetToptierLimit() int64 {
 	return c.ToptierLimit
 }
 
+func (c *container) SetPageMigration(p *PageMigrate) {
```

Comment:
nit: comment/docstring

Reply:
It is present in the interface definition. Ditto for GetPageMigration().

Reply:
Yes, of course, sorry about this